BetaUtils.Core/BetaUtils.Core.csproj

```diff
@@ -2,12 +2,18 @@
 
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
+    <LangVersion>10</LangVersion>
   </PropertyGroup>
 
   <ItemGroup>
-    <Folder Include="Extensions\" />
     <Folder Include="FeatureFlags\" />
     <Folder Include="Validation\" />
   </ItemGroup>
 
+  <ItemGroup>
+    <PackageReference Include="Hellang.Middleware.ProblemDetails" Version="3.1.0" />
+    <PackageReference Include="Microsoft.ApplicationInsights.AspNetCore" Version="2.20.0" />
+    <PackageReference Include="Microsoft.AspNetCore.Http" Version="2.2.2" />
```

Comment:
The question here would be whether we want to have things such as Http in our core library. For example, this would imply that even if we want to include this library in,  data project, we would need to install a http stuff there.

Reply:
Agree. As mention in comment above, once this logic is moved out of Code, dependencies will be gone.